lib/diagnostic.js

```diff
@@ -38,14 +49,22 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Enrique Iglesias',
+  species: 'felis catus',
+  breed: 'Exotic shorthair',
+  vocalization: 'Meow'
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
 const Pet = function() {
-  /* Your answer here */
```

Comment:
Excellent - just missing the `name`, `species`, `breed`, and `vocalization` parameters. 

For example, in the case of `this.name = name;`, `this.name` sets a `name` property to the `Pet` constructor. 

The `= name` part  - where does that `name ` come from? That should be a hint that you're expecting a name passed in as a parameter.